specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/disk.json

```diff
@@ -1647,6 +1647,10 @@
         "tier": {
           "type": "string",
           "description": "Performance tier of the disk (e.g, P4, S10) as described here: https://azure.microsoft.com/en-us/pricing/details/managed-disks/. Does not apply to Ultra disks."
+        },
+        "burstingEnabled": {
+          "type": "boolean",
+          "description": "Disk Bursting property can be used to enable support for burst beyond the provisioned perf target, up to 30K IOPS. Bursting is disabled by default. Does not apply to Ultra disks."
```

Comment:
"Set to true to enable bursting beyond the provisioned performance target of the disk."

Avoid mentioning numbers like 30 K which can change. It is better to add an aka.ms link to the paid bursting documentation when it is available.